test/infra/infra-dbdeployer-mysql56-single/docker/entrypoint.sh

```
@@ -0,0 +1,127 @@
+    -c max_connections=500 \
+    -c innodb_buffer_pool_size=128M \
+    -c innodb_log_file_size=32M \
+    -c innodb_flush_log_at_trx_commit=2 \
+    -c sync_binlog=0 \
+    -c gtid_mode=ON \
+    -c enforce_gtid_consistency=true \
+    -c ssl-ca=/root/certs/ca.pem \
+    -c ssl-cert=/root/certs/server-cert.pem \
+    -c ssl-key=/root/certs/server-key.pem
+
+SANDBOX_DIR=$(ls -d /root/sandboxes/msb_* | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)